keras_nlp/models/gpt2/gpt2_presets.py

```diff
@@ -154,4 +154,31 @@
         "merges_url": "https://storage.googleapis.com/keras-nlp/models/gpt2_base_en_news/v1/merges.txt",
         "merges_hash": "75a37753dd7a28a2c5df80c28bf06e4e",
     },
+    "gpt2_base_cn_poetry": {
```

Comment:
Is cn_poetry an official name? Otherwise "zh" might be a better bet. That would match `bert_base_zh`.

When we have talked language codes before, we have talked roughly following https://en.wikipedia.org/wiki/List_of_Wikipedias

Reply:
yea zh is the right one, changed